cli/src/clients/datafusion_helpers.rs

```diff
@@ -111,10 +111,17 @@ fn value_as_u64_opt(batch: &RecordBatch, col: usize, row: usize) -> Option<u64>
 }
 
 fn value_as_dt_opt(batch: &RecordBatch, col: usize, row: usize) -> Option<chrono::DateTime<Local>> {
-    batch
-        .column(col)
-        .as_primitive::<arrow::datatypes::Date64Type>()
-        .value_as_local_datetime_opt(row)
+    let col = batch.column(col);
+    match col.data_type() {
+        arrow::datatypes::DataType::Timestamp(arrow::datatypes::TimeUnit::Millisecond, _) => col
+            .as_primitive::<arrow::datatypes::TimestampMillisecondType>()
+            .value_as_local_datetime_opt(row),
```

Comment:
the underlying data is always utc epoch seconds, the timezone information is schema-only. i see no reason we should care about the servers timezone, we should display times in local timezone where possible

Reply:
Then I guess this needs a small follow-up because `value_as_local_datetime_opt` seems to assume utc as the timezone w/o respecting what `Timestamp` might say.

Reply:
no, its behaviour is correct i think. it just takes the numeric value (ignoring the schema timezone) as a utc epoch counter. this is correct

Reply:
let me be clear - the numeric value is the same irrelevant of what the timezone in the schema is

Reply:
True, you are right. Thanks for clarifying this for me!